app/containers/Transactions/index.jsx

```diff
@@ -54,14 +54,18 @@ export class Transactions extends React.Component { // eslint-disable-line react
       );
     } else if (!this.props.loading && (this.props.transactions.transactions || []).length === 0) {
       content = (
-        <StyledH3 className="lead text-center">No transactions found</StyledH3>
+        <StyledH3 className="lead text-center">
+          No Omni Protocol transactions found
+          <br/>
+          <a href="https://www.blocktrail.com">blocktrail.com</a>
```

Comment:
Mmm we should not just blindly add in blocktrail link. We should give a little more explanation maybe something like: 

No Omni Protocol Transactions found</h3>
If the transaction you are searching for was just broadcast it might take a few minutes for the network to pass it around for us to see it. 
If the transaction you are searching for is a Bitcoin only transaction you should use a bitcoin block explorer like blocktrail.com

Reply:
@marvgmail could you give us the explanation to add?

Reply:
Adam's words are good.

For a tx, I would make the blocktrail.com URL point to that particular tx. For an address, it should point to that address.